src/plugin/GoogleCalendar/EventSyncWindow.cpp

```diff
@@ -206,22 +225,41 @@ EventSyncWindow::_SaveSyncData(bool status)
 	time_t syncTime = time(NULL);
 
 	if (file->InitCheck() == B_OK) {
+		_SetStatusMessage(B_TRANSLATE("SaveSyncData"));
```

Comment:
I think we shouldn't mix end user and developer terms.
```suggestion
		_SetStatusMessage(B_TRANSLATE("Saving sync data"));
```